bears/general/LineBreakBear.py

```diff
@@ -0,0 +1,133 @@
+from queue import Queue
+
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.RangeHelpers import get_specified_block_range
```

Comment:
yes sure it would need a separate commit on coala for the functionality and then the bear could use that as soon as it is merged. That would make the bear incopmpatible with earlier versions of coala though. To me it makes more sense though than keeping cross-bear-functionality in coala bears.